pkgs/tools/typesetting/asciidoctor/update.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-rm gemset.nix Gemfile.lock
```

Comment:
If we do this before building the shell then we can't build the shell!

Reply:
Agreed! The more general issue is that the updater script is depending on the project to be working. So as soon as it breaks it becomes harder to repair it.

Reply:
I don't see a way to avoid this while still piggybacking off the app definition to get all the dependencies....

Reply:
:man_facepalming: probably I was running a nix-shell inside nix-shell... I wish there was `bundix --force-update`